contracts/modules/andromeda-date-time/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "andromeda-date-time"
+version = "1.0.0"
+edition = "2021"
+rust-version = "1.75.0"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[features]
+# for more explicit tests, cargo test --features=backtraces
+backtraces = ["cosmwasm-std/backtraces"]
+# use library feature to disable all instantiate/execute/query exports
+library = []
+testing = ["cw-multi-test", "andromeda-testing"]
+
+
+[dependencies]
+cosmwasm-std = { workspace = true }
+cosmwasm-schema = { workspace = true }
+cw-storage-plus = { workspace = true }
+cw-utils = { workspace = true }
+
+andromeda-std = { workspace = true, features = [] }
```

Comment:
I think `, features = []` can be removed

Reply:
resolved